### PR TITLE
[EdgeTPU] Add EdgeTPUToolchain

### PR DESCRIPTION
**Pull request by rhqjatn2398**
This PR adds EdgeTPUToolchain and the testing codes.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

### CODE REVIEW

File: src/Backend/EdgeTPU/EdgeTPUToolchain.ts [hunk 1]
```diff
@@ -140,4 +143,30 @@ class EdgeTPUCompiler extends DebianCompiler {
...
 }
 
-export { EdgeTPUDebianToolchain, EdgeTPUCompiler };
+class EdgeTPUToolchain implements Backend {
+  private readonly backendName: string;
+  private readonly toolchainCompiler: Compiler | undefined;
+
+  constructor() {
+    this.backendName = "EdgeTPU";
+    this.toolchainCompiler = undefined;
```
Comment (profornnan) [inline src/Backend/EdgeTPU/EdgeTPUToolchain.ts:152]: I think this line should be changed to use EdgeTPUCompiler

```suggestion
    this.toolchainCompiler = new EdgeTPUCompiler();
```

File: src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts
Comment (profornnan) [inline src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts:1]: Could you please move this teardown function to the bottom of this "Backend" test suite? (related to https://github.com/Samsung/ONE-vscode/issues/1678)

https://github.com/Samsung/ONE-vscode/blob/0107bfb5fb91d35442e43f22d710e8176ab232db/src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts#L59-L61

File: src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts [hunk 2]
```diff
@@ -120,151 +123,180 @@ suite("Backend", function () {
...
-      const toolchainType = "latest";
-      const start = 0;
-      const count = 0;
-      assert.deepStrictEqual(
-        edgeTPUCompiler.getToolchains(toolchainType, start, count),
-        []
-      );
+  suite("EdgeTPUCompiler", function () {
+    suite("#constructor()", function () {
+      test("Create EdgeTPUToolchain compiler", function (pass) {
```
Comment (profornnan) [inline src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts:129]: How about moving the EdgeTPUCompiler suite outside of the Backend suite?

File: src/Backend/EdgeTPU/EdgeTPUToolchain.ts (hunk 1 above)
Comment (rhqjatn2398) [reply src/Backend/EdgeTPU/EdgeTPUToolchain.ts:152]: Thanks, I'll fix the code. 😄

File: src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts
Comment (rhqjatn2398) [reply src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts:1]: Oh, I forgot that. 💦 
I'll move the teardown function to the bottom.

File: src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts [hunk 3]
```diff
@@ -267,4 +270,33 @@ suite("EdgeTPUCompiler", function () {
...
+        const edgeTPUBackend = new EdgeTPUToolchain();
+        const edgeTPUCompiler = edgeTPUBackend.compiler();
+        assert.instanceOf(edgeTPUCompiler, EdgeTPUCompiler);
+      });
+    });
+  
+    // TODO
+    // Add test case for executor() and executors()
+  });
+});
```
Comment (dayo09) [inline src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts:302]: Could you add a newline here?

**Review by dayo09 — APPROVED** [review]
LGTM :-D

**Review by profornnan — APPROVED** [review]
LGTM 😄

**Pull request merged**